scripts/e2e-test.sh

```diff
@@ -64,9 +66,13 @@ while [ "$#" -gt 0 ]; do
       help
       exit 0
       ;;
-    -l|--logs)
+    --logs)
```

Comment:
Is there a reason that the short form has been removed?

Reply:
yes because now we have `--logs` and  `--logsdir` the short form is ambiguous.